javascript.js

```diff
@@ -0,0 +1,56 @@
+const array = [];
+
+function addBook() {
+  const title = document.getElementById('title_input');
+  const author = document.getElementById('author_input');
+  if (title.value !== '' && author.value !== '') {
```

Comment:
- [ ] To follow JavaScript best practices please kindly note that `null` is also a `false` value. 

You could write it like this.
```
if(title.value && author.value)
```

meaning if title.value and author.value is true (it's not null) execute the code.